### DISCUSSION TITLE
auto tune for quad

### DISCUSSION THREAD

**Discussion by Minecraftman04** [iNavFlight/inav#8293] [Ideas]
## Current Behavior
no autotune/drone has too higher pids for weight but i have no clue on how to properly tune a pid controller. 

## Desired Behavior
auto tuning for best pid loop

## Suggested Solution
im not smart enough to fix this

## Who does this impact? Who is this for?
everyone with a quad

**Comment by erstec** [comment]
It was already discussed here https://github.com/iNavFlight/inav/discussions/7147
